primitives/application-crypto/src/lib.rs

```diff
@@ -328,6 +328,14 @@ macro_rules! app_crypto_public_common {
 	};
 }
 
+#[doc(hidden)]
+pub mod module_format_string_prelude {
+	#[cfg(all(not(feature = "std"), feature = "serde"))]
+	pub use super::{format, String};
+	#[cfg(feature = "std")]
```

Comment:
Why not just add the `std` case below [line 37](https://github.com/paritytech/substrate/blob/06d8934fe3a10f5f02b930dd14e4e3bf4baf21b0/primitives/application-crypto/src/lib.rs#L37)?

Reply:
Somewhat done in: https://github.com/paritytech/substrate/pull/14263/commits/83ea0221786c39b79f123479a34241e6d536ceff